docs/docs/getting-started/web.mdx

```diff
@@ -8,9 +8,12 @@ slug: /getting-started/web
 import {Snack} from '@site/src/components/Snack';
 
 React Native Skia runs in a web browser thanks to [CanvasKit](https://skia.org/docs/user/modules/canvaskit/), a WebAssembly build of Skia.
-The WebAssembly file is loaded asynchronously and has a size of 7.2MB.
+The WebAssembly file is loaded asynchronously and has a size of 2.9MB gzipped.
 While this is a substantial file size, you have control over the user experience: you can decide when to load Skia and how the loading experience should be.
 
+We are exploring the possibility of providing alternative versions of CanvasKit with smaller file sizes.
```

Comment:
In the end I also so think we should remove it. We're running on such thing resources that if this topic should be our focus, it would become self apparent.